File: src/ClimbingMap.Domain/Entities/Area.cs
using System.Collections.Generic;

namespace ClimbingMap.Domain.Entities {
   public class Area {
      public string Id { get; set; }
      public string Name { get; set; }
      public string Info { get; set; }
      public string Approach { get; set; }
      public string Descent { get; set; }
      public string History { get; set; }
      public string Ethics { get; set; }
      public string Accomodations { get; set; }
      public string Restrictions { get; set; }
      public string[] Tags { get; set; } = new string[0];
      public Location[] Location { get; set; } = new Location[0];
      public string[] Links { get; set; } = new string[0];
   }
}

File: src/web/boulder-guide/src/app/domain/Area.ts
import { PointOfInterest } from "./PointOfInterest";
import { Track } from "./Track";
import { Location } from "./Location";

export class Area {
   Id!: string;
   Name!: string;
   Info!: string;
   Access!: string;
   History!: string;
   Ethics!: string;
   Accommodations!: string;
   Restrictions!: string;
   Tags!: string[];
   Location!: Location[];
   Links!: string[];
   POIs!: PointOfInterest[];
   Tracks!: Track[];
}
